modules/serverless-rn-1-11-0.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-11-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.11.0
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+* Knative Eventing on {ServerlessProductName} is now GA.
+* {ServerlessProductName} Functions is now delivered as a Developer Preview through the standard Knative `kn` CLI installation. This feature provides CLI commands that developers can use to write functions using predefined runtimes and languages, including Quarkus, Node.js and Go. Functions can then be triggered using CloudEvents and plain HTTP requests.
++
```

Comment:
Maybe it would make sense to add a note about Kafka? It's now possible to install Knative Kafka via our Serverless operator. No need to install the community Knative Kafka operator.